### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by chriskim06**
Noticed there wasn't a favicon in the docs. Not sure if this is wanted, if not feel free to close

/area docs
/cc @ahmetb 
/cc @corneliusweig

### CODE REVIEW

File: site/layouts/partials/header.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     </title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="{{"/css/style.css" | relURL}}" />
+    <link rel="icon" href="{{"/img/favicon.svg"}}" type="image/svg" />
```
Comment (ahmetb) [inline site/layouts/partials/header.html:17]: can you follow the `| relURL ` method as above. that basically ensures that if we someday host the site on /site, we can still have /site/img/favicon

**Comment by ahmetb** [comment]
Seems to be working on preview. 👍🏼 One nit.

**Comment by ahmetb** [comment]
/lgtm
/approve
thanks.

**Comment by corneliusweig** [comment]
Nice, thanks!

**Pull request merged**